libnmstate/nm/ip.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import socket
+
+from . import nmclient
+
+
+def get_ip_info(active_connection, is_ipv6=False):
```

Comment:
Boolean flags imply that this function is doing more than one thing.
It is preferred to use separated functions.
The common code can be called from both such functions.